middleware.js

```diff
@@ -49,6 +61,24 @@ module.exports.fetchRandomGameData = async (req, res, next) => {
   }
 };
 
+module.exports.fetchRandomGameDataArr = async (req, res, next) => {
+  const pageNumArr = generateUniqueRandomArr();
+  let gameDataArr = [];
+  for (let j = 0; j < 5; j++) {
+    response = await axios.get(
+      `${apiUrl}games?key=${process.env.RAWG_KEY}&page=${pageNumArr[j]}&${apiParams}`
+    );
+    for (let i = 0; i < response.data.results.length; i++) {
+      gameDataArr.push({
+        name: response.data.results[i].name,
+        image: response.data.results[i].background_image,
```

Comment:
What happens in case the key value of `background_image` is null or undefined?

Reply:
The app will crash. Forgot to consider this point, thanks for pointing it out, will fix it